spec/beaker/hypervisor/docker_spec.rb

```diff
@@ -683,48 +682,48 @@ module Beaker
+                                                                  'image' => 'foobar',
+                                                                  'docker_image_entrypoint' => '/bin/bash',
+                                                                }))
 
             expect(dockerfile).to match(%r{ENTRYPOINT /bin/bash})
           end
         end
 
         it 'uses zypper on sles' do
           FakeFS.deactivate!
-          dockerfile = docker.send(:dockerfile_for, {
-                                     'platform' => 'sles-12-x86_64',
-                                     'image' => 'foobar',
-                                   })
+          dockerfile = docker.send(:dockerfile_for, make_host('none', {
+                                                                'platform' => Beaker::Platform.new('sles-12-x86_64'),
+                                                                'image' => 'foobar',
+                                                              }))
 
           expect(dockerfile).to match(/zypper -n in openssh/)
         end
 
         (22..39).to_a.each do |fedora_release|
           it "uses dnf on fedora #{fedora_release}" do
             FakeFS.deactivate!
-            dockerfile = docker.send(:dockerfile_for, {
-                                       'platform' => "fedora-#{fedora_release}-x86_64",
-                                       'image' => 'foobar',
-                                     })
+            dockerfile = docker.send(:dockerfile_for, make_host('none', {
+                                                                  'platform' => Beaker::Platform.new("fedora-#{fedora_release}-x86_64"),
+                                                                  'image' => 'foobar',
+                                                                }))
 
             expect(dockerfile).to match(/dnf install -y sudo/)
           end
         end
 
         it 'uses pacman on archlinux' do
           FakeFS.deactivate!
-          dockerfile = docker.send(:dockerfile_for, {
-                                     'platform' => 'archlinux-current-x86_64',
-                                     'image' => 'foobar',
-                                   })
+          dockerfile = docker.send(:dockerfile_for, make_host('none', {
+                                                                'platform' => Beaker::Platform.new('archlinux-current-x86_64'),
+                                                                'image' => 'foobar',
+                                                              }))
 
           expect(dockerfile).to match(/pacman --sync --refresh --noconfirm archlinux-keyring/)
           expect(dockerfile).to match(/pacman --sync --refresh --noconfirm --sysupgrade/)
-          expect(dockerfile).to match(/pacman --sync --noconfirm curl ntp net-tools openssh/)
+          expect(dockerfile).to match(/pacman --sync --noconfirm curl net-tools openssh/)
```

Comment:
we could make this
```suggestion
          expect(dockerfile).to match(/pacman --sync --noconfirm curl/)
```
and then it'd pass on 6.3 AND 6.4